4-WebApp-your-API/4-1-MyOrg/README.md

```diff
@@ -1,225 +1,269 @@
+    git clone https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2.git
 ```
 
 or download and extract the repository .zip file.
 
-> :warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
+>:warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
 
-### Register the sample application(s) with your Azure Active Directory tenant
+### Step 2: Navigate to project folder
+```console
+    cd 4-WebApp-Your-API\4-1-MyOrg
+```
 
-There are two projects in this sample. Each needs to be separately registered in your Azure AD tenant. To register these projects, you can:
+### Step 3: Application Registration
 
-- follow the steps below for manually register your apps
-- or use PowerShell scripts that:
-  - **automatically** creates the Azure AD applications and related objects (passwords, permissions, dependencies) for you.
-  - modify the projects' configuration files.
 
-<details>
-  <summary>Expand this section if you want to use this automation:</summary>
+There are two projects in this sample. Each needs to be separately registered in your Azure AD tenant. To register these projects, you can:
 
-> :warning: If you have never used **Azure AD Powershell** before, we recommend you go through the [App Creation Scripts](./AppCreationScripts/AppCreationScripts.md) once to ensure that your environment is prepared correctly for this step.
+- follow the steps below for manually register your apps 
+  <details>
+   <summary>Expand this section to view manual steps:</summary>
+      
+   1. Sign in to the [Azure portal](https://portal.azure.com).
+   1. If your account is present in more than one Azure AD tenant, select your profile at the top right corner in the menu on top of the page, and then **switch directory** to change your portal session to the desired Azure AD tenant.
+              
+   ### Register the service app (TodoListService-aspnetcore-webapi)
+
+   1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure AD** service.
+   1. Select the **App Registrations** blade on the left, then select **New registration**.
+   1. In the **Register an application page** that appears, enter your application's registration information:
+      - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `TodoListService-aspnetcore-webapi`.
+   1. Under **Supported account types**, select **Accounts in this organizational directory only**
+   1. Click **Register** to create the application.
+
+      ![AppRegistrationMain](./ReadmeFiles/AppRegistrationMainScreen.png)
+   
+  
+   1. In the app's registration screen, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+
+   1. Select **ID tokens (used for implicit and hybrid flows)** checkbox.
```

Comment:
> 1. Select **ID tokens (used for implicit and hybrid flows)** checkbox. [](http://example.com/codeflow?start=2&length=71)

this is not required for a Web API at all??

Reply:
removed